### ISSUE TITLE
Canada and US number Confusion

### ISSUE THREAD

**Issue by supermarrk** [nikhiln/ngx-international-phone-number#3]
Hello,

I am using Angular 4 and I implemented this plugin in my project.
My issue is when I am retrieving a Canadian mobile number the flag is set to US flag.

![image](https://user-images.githubusercontent.com/35980809/37893696-68e9bed4-310e-11e8-8584-4eb9001b3820.png)

Thanks,

**Comment by sonamgupta-kiwi** [comment]
any update on this issue??

**Comment by BrianCerasuolo** [comment]
canada and us share the same country code

**Comment by andrey-o-sim** [comment]
@nikhiln 
Yes, these countries has the same country code. But that's wrong behavior, when I manually select "Canada"   and then type phone number and flag are updated to US. The flag shouldn't be changed in this case. Only in case if I provide another country code.
Please, fix this issue, because this is critical bug.
Thanks

**Comment by quekqh** [comment]
hi any updates on this?

**Comment by ShubhJaiswal** [comment]
@nikhiln 
You can externally install 'libphonenumber-js' and you can parse your number, so there you will get your country initial that you can pass , Your canada and us issue can be resolved, i did same way

**Comment by choopage** [comment]
This might have been fixed by https://github.com/esoyke/ngx-international-phone-number2
